homeassistant/components/speedtestdotnet/sensor.py

```diff
@@ -43,19 +45,35 @@ class SpeedtestSensor(CoordinatorEntity, RestoreEntity, SensorEntity):
     """Implementation of a speedtest.net sensor."""
 
     coordinator: SpeedTestDataCoordinator
+    entity_description: SpeedtestSensorEntityDescription
     _attr_icon = ICON
 
     def __init__(
         self,
         coordinator: SpeedTestDataCoordinator,
-        description: SensorEntityDescription,
+        description: SpeedtestSensorEntityDescription,
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
         self.entity_description = description
         self._attr_name = f"{DEFAULT_NAME} {description.name}"
         self._attr_unique_id = description.key
         self._attrs = {ATTR_ATTRIBUTION: ATTRIBUTION}
+        self._attr_device_info = {
+            "identifiers": {(DOMAIN, "Speed Test")},
```

Comment:
This means only a single speed test device entry will be possible, that's intentional since we only allow a single config entry?

Reply:
We should still just limit it to config entry ID.

Reply:
OK, so: `"identifiers": {(DOMAIN, config_entry.entry_id)},`, in that way multiple config entries can be supported in the future

Reply:
Yes, if there is no other unique identifier, that will do the trick.